source/usr/lib/mcvirt/parser.py

```diff
@@ -1201,3 +1224,16 @@ def parse_arguments(self, script_args=None):
                 user = user_factory.get_user_by_username(args.remove_username)
                 rpc.annotate_object(user)
                 user.delete()
+        elif action == 'vnc':
```

Comment:
It's normally quite nice to put this code in the position relative to the parser order (so next to the 'info' handling code.